drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -220,6 +222,16 @@ int i2c_stm32_target_register(const struct device *dev,
 		return ret;
 	}
 
+#if defined(CONFIG_PM_DEVICE_RUNTIME)
+	if (pm_device_wakeup_is_capable(dev)) {
+		/* Mark device as active */
+		(void)pm_device_runtime_get(dev);
```

Comment:
I'm not familiar with I2C target API, but `pm_device_runtime_get` is already called in `i2c_configure`, isn't it sufficient ?

Reply:
The call in `stm32_i2c_runtime_configure` is follow by a call to `pm_device_runtime_put`. For wake up from STOP to work, we need to keep the peripheral active as long as the target mode is configured. At least on L0 the peripheral clock needs to stay on for the address match detection to work. From experience, I believe the peripheral clock can be turned off on L4, but I would have test again to verify.

Reply:
ok, that's looks fine to me.